python/splitter.py

```diff
@@ -20,6 +20,21 @@ def is_number(string):
     except ValueError:
         return False
 
+def validate_input(prompt):
+    #This function will take in a question prompt for an input and check to make sure it is a valid number.
```

Comment:
Programming languages have specific ways to write documentation. Python has docstrings https://www.python.org/dev/peps/pep-0257/